modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -503,6 +503,118 @@ module_stream_v2_test_parse_dump (ModuleStreamFixture *fixture,
     "...\n");
 }
 
+static void
+module_stream_v1_test_depends_on_stream (ModuleStreamFixture *fixture,
+                                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  g_autofree gchar *path = NULL;
+  g_autoptr (GError) error = NULL;
+  g_autofree gchar *module_name = NULL;
+  g_autofree gchar *module_stream = NULL;
+
+  path = g_strdup_printf ("%s/modulemd/v2/tests/test_data/dependson_v1.yaml",
+                          g_getenv ("MESON_SOURCE_ROOT"));
+  g_assert_nonnull (path);
+  stream = modulemd_module_stream_read_file (
+    path, TRUE, module_name, module_stream, &error);
+  g_assert_nonnull (stream);
+  g_assert_true (
+    modulemd_module_stream_depends_on_stream (stream, "platform", "f30"));
+  g_clear_object (&stream);
+
+  stream = modulemd_module_stream_read_file (
```

Comment:
You don't need to read the file again. You can perform this test with the same `ModulemdModuleStream` object you used for line 523.

Reply:
Please fix this for all of the tests below and in the v2 version. After that, this should be good to merge.